docs/how_to/detect_and_annotate.md

```diff
@@ -1 +1,63 @@
-🚧 Page under construction.
+## How to Detect and Annotate
+
+Supervision enables you to annotate predictions from object detection and segmentation models in a few lines of code.
+
+The `sv.BoxAnnotator()` class lets you annotate images with bounding boxes, while the `sv.MaskAnnotator()` class lets you annotate images with segmentation masks. `sv.MaskAnnotator()` is a drop-in replacement for `sv.BoxAnnotator()`, so you don't need to update any other code if you want to switch between the two.
+
+In this guide, we run inference on an Ultralytics YOLOv8 object detection model and plot the predictions with the `sv.BoxAnnotator()` class.
+
+## Load a Model and Retrieve Predictions
+
+First, we need to retrieve predictions from a model. For this guide, we will retrieve predictions from a YOLOv8 model. See a list of all models from which you can load predictions into supervision.
+
+```python
+import supervision as sv
+from ultralytics import YOLO
+import cv2
+
+model = YOLO("yolov8n.pt")
+
+results = model("image.jpg")
+```
+
+In this code, we import `supervision`, `ultralytics`, and `cv2` (OpenCV). We then load a YOLOv8 object detection model, then run inference on an image called `image.jpg`. The `results` variable contains the predictions from the model.
+
+## Load Predictions into Supervision
```

Comment:
Please mention other connectors, especially `from_roboflow`. We probably need to use YOLOv8 as an example, but let's show other options.